includes/models/model.llms.quiz.php

```diff
@@ -108,6 +109,26 @@ public function get_questions( $return = 'questions' ) {
 		return $this->questions()->get_questions( $return );
 	}
 
+	/**
+	 * Get questions count.
+	 *
+	 * @since [version]
+	 *
+	 * @return int Question Count.
+	 */
+	public function get_questions_count() {
+
+		/**
+		 * Filter the count of questions in a quiz.
+		 *
+		 * @since [version]
+		 *
+		 * @param int       $questions_count Number of questions in a quiz.
+		 * @param LLMS_Quiz $this            Current quiz object.
```

Comment:
In this case use `$quiz` instead of `$this` in the docblock.
This docblock is going to create an actual documentation, e.g. https://developer.lifterlms.com/reference/hooks/llms_student_query_default_args/

Reply:
got it.